apps/reference/docs/guides/hosting/docker.mdx

```diff
@@ -95,6 +95,11 @@ POSTGRES_PORT=5432
 
 Once you have done this, you can safely comment out the `db` section of the `docker-compose` file, and remove any instances where the services `depends_on` the `db` image.
 
+### Setting database's `log_min_messages`
+
+`docker-compose` sets the database's `log_min_messages` configuration to `fatal` because it prevents a lot of redundant logs generated by Realtime.
+However, you might miss important log messages, such as database errors, by default. Please remember to configure `log_min_messages` based on your needs.
```

Comment:
```suggestion
By default, `docker-compose` sets the database's `log_min_messages` configuration to `fatal` to prevent redundant logs generated by Realtime.
However, you might miss important log messages such as database errors. Configure `log_min_messages` based on your needs.
```